### PR TITLE
feat(18-dependency-init): Display error message when dependency init failed

### LINKED ISSUE #18: Catch dependency initialization exceptions at startup
This will be initialized at app startup but I think if the app can't retrieve the documentsDirectory there no point to even try to display anything other than an error screen saying `Well something fucked up so restart the app please`.
So we could just `try-catch` the `configureDependencies` method in `main` and if it fails display said error message.
If it succeeds we'll launch the app normally

_Originally posted by @ronw98 in https://github.com/MovnaTeam/movna/pull/14#discussion_r1289048622_

### PR DESCRIPTION
**Pull request by ronw98**
Improve dark mode

| App dark | App Light | Init failed light | Init failed dark
| ------ | ----- | ---- | ------- 
| image | image| image |  image

Closes #18

### CODE REVIEW

File: assets/translations/en.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "appTitle": "Movna",
+  "broken_app": {
+    "button": "Restart the app",
+    "label": "It looks like the app crashed while launching, you can try to restart it"
```
Comment (BriceCroix) [inline assets/translations/en.json:5]:
```suggestion
    "label": "It looks like the app crashed while launching, you can try to restart it."
```

File: assets/translations/fr.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
   "appTitle": "Movna",
+  "broken_app": {
+    "button": "Redémarrer l'application",
+    "label": "L'application a rencontré un problème au lancement. Essayez de la redémarrer"
```
Comment (BriceCroix) [inline assets/translations/fr.json:5]:
```suggestion
    "label": "L'application a rencontré un problème au lancement. Essayez de la redémarrer."
```

**Review by BriceCroix — APPROVED** [review]
Great work there, I really like the idea of the brokenApp and the themed svg !

**Comment by ronw98** [comment]
> Great work there, I really like the idea of the brokenApp and the themed svg !

Thanks ! I full on copied the themed svg code from my job though 😆

**Pull request merged**